codalab/apps/web/migrations/0060_auto__add_field_competitionphase_scoring_program_docker_image__add_fie.py

```diff
@@ -1,478 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
So all of these migrations were deleted? Can we confirm migrations/such from prod database will work OK against these changes?

Reply:
It doesn't look like old migrations/db schema will work with these ones without some work.

I did the following:
- reset the db
- switched to develop & ran migrations
- switched back to this branch
- faked migrations

Tried to create a user and started getting some errors from Django:
```
django            | 2020-07-22 22:27:15,151 ERROR Internal Server Error: /accounts/signup/
django            | Traceback (most recent call last):
django            |   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 41, in inner
django            |     response = get_response(request)
django            |   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 187, in _get_response
django            |     response = self.process_exception_by_middleware(e, request)
django            |   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 185, in _get_response
django            |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
django            |   File "/usr/local/lib/python3.8/site-packages/django/views/generic/base.py", line 68, in view
django            |     return self.dispatch(request, *args, **kwargs)
django            |   File "/usr/local/lib/python3.8/site-packages/django/utils/decorators.py", line 67, in _wrapper
django            |     return bound_func(*args, **kwargs)
django            |   File "/usr/local/lib/python3.8/site-packages/django/views/decorators/debug.py", line 76, in sensitive_post_parameters_wrapper
django            |     return view(request, *args, **kwargs)
django            |   File "/usr/local/lib/python3.8/site-packages/django/utils/decorators.py", line 63, in bound_func
django            |     return func.__get__(self, type(self))(*args2, **kwargs2)
django            |   File "/usr/local/lib/python3.8/site-packages/allauth/account/views.py", line 214, in dispatch
django            |     return super(SignupView, self).dispatch(request, *args, **kwargs)
django            |   File "/usr/local/lib/python3.8/site-packages/allauth/account/views.py", line 77, in dispatch
django            |     response = super(RedirectAuthenticatedUserMixin,
django            |   File "/usr/local/lib/python3.8/site-packages/allauth/account/views.py", line 190, in dispatch
django            |     return super(CloseableSignupMixin, self).dispatch(request,
django            |   File "/usr/local/lib/python3.8/site-packages/django/views/generic/base.py", line 88, in dispatch
django            |     return handler(request, *args, **kwargs)
django            |   File "/usr/local/lib/python3.8/site-packages/allauth/account/views.py", line 103, in post
django            |     response = self.form_valid(form)
django            |   File "/usr/local/lib/python3.8/site-packages/allauth/account/views.py", line 230, in form_valid
django            |     self.user = form.save(self.request)
django            |   File "/usr/local/lib/python3.8/site-packages/allauth/account/forms.py", line 407, in save
django            |     setup_user_email(request, user, [])
django            |   File "/usr/local/lib/python3.8/site-packages/allauth/account/utils.py", line 275, in setup_user_email
django            |     a.save()
django            |   File "/usr/local/lib/python3.8/site-packages/django/db/models/base.py", line 762, in save
django            |     raise ValueError(
django            | ValueError: save() prohibited to prevent data loss due to unsaved related object 'user'.
```

It could be from differences in migrations from different versions of allauth, but that's just a guess since the error seems to relate to allauth in some way.